extensions/io/org.eclipse.smarthome.io.mqttembeddedbroker/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,27 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Embedded Mqtt Broker
+Bundle-SymbolicName: org.eclipse.smarthome.io.mqttembeddedbroker;singleton:=true
+Bundle-Vendor: Eclipse.org/SmartHome
+Bundle-Version: 0.9.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: ., lib/moquette-broker-0.10.jar, lib/netty-all-4.1.13.Final.jar
+Import-Package: 
+ com.google.common.collect,
+ org.apache.commons.io,
+ org.apache.commons.lang,
+ org.apache.commons.net.util,
+ org.eclipse.jdt.annotation;resolution:=optional,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.types,
+ org.eclipse.smarthome.io.transport.mqtt,
+ org.eclipse.smarthome.io.transport.mqtt.reconnect,
+ org.eclipse.smarthome.io.transport.mqtt.sslcontext,
+ org.osgi.framework;version="1.8.0",
+ org.osgi.service.component,
+ org.osgi.service.component.annotations;resolution:=optional,
```

Comment:
I assume this should be removed: https://github.com/openhab/openhab2-addons/pull/2878#discussion_r152942296